core/src/main/java/hera/core/commands/Version.java

```diff
@@ -0,0 +1,26 @@
+package hera.core.commands;
+
+import discord4j.core.event.domain.message.MessageCreateEvent;
+import discord4j.core.object.entity.Guild;
+import discord4j.core.object.entity.Member;
+import discord4j.core.object.entity.MessageChannel;
+import hera.core.HeraUtil;
+import hera.database.entities.mapped.Localisation;
+import hera.database.types.GlobalSettingKey;
+import hera.database.types.LocalisationKey;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+import static hera.store.DataStore.STORE;
+
+public class Version {
+	public static Mono<Void> execute(MessageCreateEvent event, Guild guild, Member member, MessageChannel channel, List<String> params) {
+		return channel.createMessage(getVersion(guild)).then();
+	}
+
+	private static String getVersion(Guild guild) {
+		Localisation message = HeraUtil.getLocalisation(LocalisationKey.COMMAND_VERSION, guild);
+		return String.format(message.getValue(), STORE.globalSettings().forKey(GlobalSettingKey.VERSION).get(0).getValue());
```

Comment:
You should probably check if the list of `GlobalSettings` actually contains something.
Else `.get(0)` could possibly get you a nullpointer.